deeplake/core/dataset/dataset.py

```diff
@@ -3062,7 +3063,11 @@ def extend(
                 samples, extend=True, skip_ok=skip_ok, append_empty=append_empty
             )
         with self:
-            for i in range(n):
+            if progressbar:
```

Comment:
It will be good to add feature_report to both extend and append or finally to _append_or_extend

Reply:
I think we don't report these because these calls would be too frequent